### PR TITLE
Cockpit 269

### PR DESCRIPTION
**Pull request by garrett**
Preview: https://garrett.github.io/cockpit-project.github.io/blog/cockpit-269.html

### CODE REVIEW

**Comment by garrett** [comment]
Here's a dark mode composite with dark on the left (this is the one I suggest):

cockpit-dark-left

Here it is with dark on the right:

cockpit-dark-right

Here are the source images:

Screenshot from 2022-05-12 19-11-27

Screenshot from 2022-05-12 19-11-15

[cockpit-dark.xcf.gz](https://github.com/cockpit-project/cockpit-project.github.io/files/8681321/cockpit-dark.xcf.gz)

**Comment by garrett** [comment]
Applied a lot of suggestions (kept as a separate commit for now) and added dark mode screenshot (also separate commit for now).

File: _posts/2022-05-12-cockpit-269.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+tags: cockpit, machines
+slug: cockpit-269
+category: release
+summary: 'Dark mode for Cockpit Client, metrics usage shows containers, default fallback name for new VMs'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 269 and cockpit-machines 269:
```
Comment (allisonkarlitskaya) [inline _posts/2022-05-12-cockpit-269.md:14]: It's cockpit-machines 268.

**Review by allisonkarlitskaya — APPROVED** [review]
Looks good.

We don't mention the cockpit-ostree release, but we also don't really have any notes from it.  I'll include it in the email.

Thanks!

**Pull request merged**